samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -1,13 +1,13 @@
 .. _broadcast_configuration_tool:
 
-Broadcast Configuration Tool
-############################
+Auracast: Broadcast Configuration Tool
```

Comment:
Do you need to have the TM here as well?

Reply:
It's in the first line of the doc, and from what I see for Bluetooth, the R is not used in the titles. So the TM in the first line of the doc should be enough.